### PR TITLE
[VIDCS-923] Android MP Video Transformers sample

### PR DESCRIPTION
**Pull request by goncalocostamendes**
**What is this PR doing?**

Adds Video transformers sample.

**How should this be manually tested?**

There is a button which toggles the video transformers (background blur and logo).
Once set, it should be visible instantly on the video stream.

https://jira.vonage.com/browse/VIDCS-923

### CODE REVIEW

File: Video-Transformers-Java/app/src/main/java/com/tokbox/sample/videotransformers/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+
+            // Obtain the Y  plane of the video frame
+            ByteBuffer yPlane = frame.getYplane();
+
+            // Get the dimensions of the video frame
+            int videoWidth = frame.getWidth();
+            int videoHeight = frame.getHeight();
+
+            // Get the dimensions of the logo
+            Bitmap image = BitmapFactory.decodeResource(context.getResources(), R.drawable.vonage_logo);
```
Comment (juliobecerragomez) [inline Video-Transformers-Java/app/src/main/java/com/tokbox/sample/videotransformers/MainActivity.java:311]: Does it mean we are going to run this decodeResource for every frame? What's the cost of this operation?

**Review by juliobecerragomez — APPROVED** [review]
Looks very good, just a minor comment.

File: Video-Transformers-Java/app/src/main/java/com/tokbox/sample/videotransformers/MainActivity.java (hunk 1 above)
Comment (juliobecerragomez) [reply Video-Transformers-Java/app/src/main/java/com/tokbox/sample/videotransformers/MainActivity.java:311]: BTW I would add that the comment is misleading. This is not exactly getting the dimensions of the logo.

**Comment by v-kpheng** [comment]
@goncalocostamendes, can you please update with test instructions?

File: Video-Transformers-Java/app/src/main/java/com/tokbox/sample/videotransformers/MainActivity.java (hunk 1 above)
Comment (goncalocostamendes) [reply Video-Transformers-Java/app/src/main/java/com/tokbox/sample/videotransformers/MainActivity.java:311]: we can do this on startup indeed!

**Pull request merged**